aws/cloudformation.yml

```diff
@@ -0,0 +1,699 @@
+                  echo "$?" >> /var/tmp/bootstrap.txt
+                  cd /home/mastodon/live
+                  echo "git checkout" >> /var/tmp/bootstrap.txt
+                  git checkout ${ReleaseVersion}
+                  echo "$?" >> /var/tmp/bootstrap.txt
+
+                  echo "copy AWS docker-compose.yml" >> /var/tmp/bootstrap.txt
+                  rm -f docker-compose.yml
+                  cp /var/tmp/docker-compose.yml /home/mastodon/live/docker-compose.yml
+
+                  echo "create .env.production" >> /var/tmp/bootstrap.txt
+                  cat >> /home/mastodon/live/.env.production << EOF
+                  LOCAL_DOMAIN=${Domain}
+                  SINGLE_USER_MODE=false
+                  SECRET_KEY_BASE=fc6379db2af899b3c980c83fc725f06996e9c5994b8adf34ec3c346731b6964c7b850209e0edfbc594d26a95108b6fe344bd0c9d2deeee127b51184d75a90cce
+                  OTP_SECRET=0ee36c5efb11cda1113ee59a3506003d4660fd1eeedc858391731baf9c6435dac572e0ec15474be64deeadd84df6f61c8b54070610f34f772c2e16e050587435
+                  VAPID_PRIVATE_KEY=z2es43uf_5z1z1ZiJRK1bMAeWS02UO871NGAuhBpTSU=
+                  VAPID_PUBLIC_KEY=BI_XIXxCKG1E_shNXhs_G5ehCYqml8alsUWkpTSUjNL8nDqZGcd-bUMfCKU70x5sUJxXqp7PtIGWkA6yQ6xNP00=
+                  DB_HOST=${DB_ENDPOINT}
+                  DB_PORT=${DB_PORT}
+                  DB_NAME=mastodon_production
+                  DB_USER=${RDSPostgreSQLMasterUsername}
+                  DB_PASS=${RDSPostgreSQLMasterPassword}
+                  REDIS_HOST=${REDIS_ENDPOINT}
+                  REDIS_PORT=${REDIS_PORT}
+                  REDIS_PASSWORD=
+                  S3_ENABLED=true
+                  S3_PROTOCOL=https
+                  S3_BUCKET=${UniqueID}.files.${Domain}
+                  S3_REGION=${AWS::Region}
+                  S3_HOSTNAME=s3-${AWS::Region}.amazonaws.com
+                  AWS_ACCESS_KEY_ID=${S3_ACCESSKEY}
+                  AWS_SECRET_ACCESS_KEY=${S3_SECRETKEY}
+                  SMTP_SERVER=smtp.mailgun.org
+                  SMTP_PORT=587
+                  SMTP_LOGIN=${MailgunLOGIN}
+                  SMTP_PASSWORD=${MailgunPASSWORD}
+                  SMTP_AUTH_METHOD=plain
+                  SMTP_OPENSSL_VERIFY_MODE=none
+                  SMTP_FROM_ADDRESS=${Domain} <${MailgunLOGIN}>
+                  EOF
+
+                  cd /home/mastodon/live  # Just incase
+
+                  echo "docker-compose build" >> /var/tmp/bootstrap.txt
+                  docker-compose build 
+                  echo "$?" >> /var/tmp/bootstrap.txt
+
+                  echo "db:setup" >> /var/tmp/bootstrap.txt
+                  docker-compose run --rm web rails db:setup -e SAFETY_ASSURED=1 -e RAILS_ENV=production # Create DB tables <-- BORKED?
+                  echo "$?" >> /var/tmp/bootstrap.txt
+
+                  echo "assets:precompile" >> /var/tmp/bootstrap.txt
+                  docker-compose run --rm web rails assets:precompile  # Compile assetts
```

Comment:
Bundler context:
`docker-compose run --rm web bundle exec rails assets:precompile`

Reply:
I didn't know the difference.. If the bundle is the preferred method, we should consider updating the doco to match.

I've made these changes.